test/JDBC/upgrade/14_latest/schedule

```diff
@@ -38,4 +38,5 @@ BABEL-3249
 BABEL-3204
 BABEL-3234
 constraint_column_usage
-column_domain_usage
\ No newline at end of file
+column_domain_usage
```

Comment:
this is sufficient as future upgrade test but now we do not have any thing to test from 13 to 14